gemini/use-cases/retrieval-augmented-generation/rag_vector_embedding_in_bigquery.ipynb

```diff
@@ -0,0 +1,794 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    new_conn_parent = f"projects/{PROJECT_ID}/locations/{REGION}"
</pre><p>There should be client methods to create these strings instead of hardcoding them</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/487/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>